src/client/actions/MessageReactionAdd.js

```diff
@@ -18,8 +18,10 @@ class MessageReactionAdd extends Action {
   handle(data) {
     if (!data.emoji) return false;
 
-    const user = this.getUserFromMember(data);
-    if (!user) return false;
+    const client = this.client;
+    const guild = client.guilds.cache.get(data.guild_id);
```

Comment:
There's no point in creating a separate variable for the client
```suggestion
    const guild = this.client.guilds.cache.get(data.guild_id);
```